Gopkg.toml

```diff
@@ -38,32 +38,33 @@
 
 [[constraint]]
   name = "github.com/kubernetes-csi/external-snapshotter"
-  version = "1.0.0-rc2"
+  #version = "1.0.0-rc2"
+  branch = "master"
```

Comment:
why does this need to be master?

Reply:
Ah ok didn't realize rc3 has been cut, will update